cli/asc.js

```diff
@@ -220,7 +229,7 @@ exports.main = function main(argv, options, callback) {
         : path.join(process.cwd(), filename);
       if (/\.ts$/.test(filename)) require("ts-node").register({ transpileOnly: true, skipProject: true });
       try {
-        const classOrModule = require(filename);
+        const classOrModule = require(pathResolves(filename) ? filename : transformArgs[i]);
```

Comment:
Here, `transfromArgs[i]` is not trimmed, which `filename` is guaranteed to be, and trying to require that is unnecessary if the path is absolute anyway. Also, `require(transformArgs[i])` is relative to `node_modules/assemblyscript/cli/asc.js` which appears problematic. A cleaner variant might be
* if absolute, require as-is
* if relative, use [node-resolve](https://www.npmjs.com/package/resolve) relative to cwd?

Reply:
resolve can take an array of paths to use instead of the default, so we could use cwd and baseDir there.

Also why is the filename not guaranteed to be trimmed?  Shouldn't all cli arguments be trimmed?